vulnerability-disclosure-policy.md

```diff
@@ -56,6 +56,8 @@ Reports should include:
 
 * Any technical information and related materials we would need to reproduce the issue.
 
+**Note: At this time, we do not currently support GPG-encrypted emails for vulnerability reports. For particularly sensitive information, please contact via an TLS-encrypted form.**
+
```

Comment:
I'd remove the "at this time" - it makes it sound like we want to add GPG but haven't yet. I'm not sure we do: there's a lot to suggest that the TLS-protected form is the best approach anyway.